### PR TITLE
fix: change import of @yamada-ui/lucide in disclosure of Storybook to *Icon

### LINKED ISSUE #3387: Changed import of `@yamada-ui/lucide` in `disclosure` of Storybook to `*Icon`.
### Description

Correct the writing style deprecated with this PR.
#3358

For example, fix the parts in `stories/components/data-display/calendar.stories.tsx` that are not imported in `*Icon` format.

```diff
- import { Ghost } from "@yamada-ui/lucide"
+ import { GhostIcon } from "@yamada-ui/lucide"
```

### Problem Statement/Justification

Because it's deprecated.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by ChinoUkaegbu**
Closes #3387

## Description

Changes import of @yamada-ui/lucide in disclosure of Storybook to *Icon.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by illionillion — CHANGES_REQUESTED** [review]
@ChinoUkaegbu 
Thank for creating PR!
Storybook is not a modification of the library itself, so there is no need to create a changeset.

File: .changeset/lemon-dryers-own.md
Comment (illionillion) [inline .changeset/lemon-dryers-own.md:1]: Plase remove this file.

**Comment by ChinoUkaegbu** [comment]
Got it! I removed the file

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**